### ISSUE TITLE
Only show clean / related job logs related to the test

### ISSUE THREAD

**Issue by jackjia-ibm** [zowe/zowe-install-test#155]
Regarding the listing of JES jobs, it would be useful for the pipeline to purge old jobs before starting the test.  Not urgent, but nice to have.  Alternatively, make show-job-logs.sh show only the ones from the current test (or day if that's hard to do)
